src/status_im/contexts/profile/contact/actions/view.cljs

```diff
@@ -22,6 +22,13 @@
                                                         :text (i18n/label :t/nickname-removed)}])
                                          (rf/dispatch [:contacts/update-nickname public-key ""]))
                                        [public-key])
+        on-show-qr                    (rn/use-callback
```

Comment:
I'm always confused of whether we actually need to use `use-callback` when there is no dependencies needed. Do we really gain from this approach. 
I saw @cammellos comment on this recently and I personally don't think we should unless there is a specific need for that page/component to make use of (use-callback with no dependencies needed).